src/EnergyPlus/WindowComplexManager.cc

```diff
@@ -958,35 +959,6 @@ namespace WindowComplexManager {
         }
     }
 
-    void DetermineMaxBackSurfaces(EnergyPlusData &state)
```

Comment:
Interesting that this function was present but never used ever since BSDF was added.